app/react/utils/api.js

```diff
@@ -20,15 +20,58 @@ const doneLoading = data => {
 
 const isNonUsualApiError = error => error.status && ![401, 404, 409, 500].includes(error.status);
 
+const errorMessages = [
+  {
+    key: /PayloadTooLargeError: request entity too large/g,
+    message: 'The request has too large data. Please review any long value property.',
```

Comment:
Sorry but again this is not correct English. :(

“Request data too large.  Please review text length in property values”